### ISSUE TITLE
Unexpected Behavior with ActsAsTaggableOn

### ISSUE THREAD

**Issue by romanos** [ForestAdmin/forest-rails#247]
## Expected behavior

My app makes use of ActsAsTaggableOn gem for certain models. I expect that behavior to remain unchanged with the integration of Forest.

## Actual Behavior

Forest makes every model in my app Taggable, regardless of whether or not I have specified such behavior. This is **really** problematic in high-write models, where ActsAsTaggableOn performs a SELECT query after every single insert. This has created timeouts and performance bottlenecks around my app.

## Replication

This was a really challenging issue to isolate, so I just created a blank Rails app from scratch and added Forest Admin to the Gemfile and created the initializer file. 

https://github.com/jargon-ai/tagging_test

Note: Car & Truck models exist. Car is a taggable, Truck is not. 

1. Boot up rails console, `rails c`
2. `Car.create` should generate an INSERT query followed by a SELECT query for Taggable
3. `Truck.create` should generate **only** the INSERT query. It is not taggable. Instead, this generates identical behavior to Step 2.
4. `truck = Truck.create`, this new `truck` object responds to `truck.tags` and other ActsAsTaggable convenience methods.
5. Remove `forest_liana` from Gemfile and disable initializer file, behavior reverts to expected. 

@SeyZ @louisteenbrink let me know if you need more info. This bug killed us in production 😢

**Comment by romanos** [comment]
I don't know where the problem stems, but I'm assuming around here:

From [forest-rails/app/deserializers/forest_liana/resource_deserializer.rb](https://github.com/ForestAdmin/forest-rails/blob/0960413dd1927719401f3bc6d71d4bba4a83e826/app/deserializers/forest_liana/resource_deserializer.rb#L188) 

```ruby
def has_acts_as_taggable?
  @resource.respond_to?(:acts_as_taggable) &&
    @resource.acts_as_taggable.respond_to?(:to_a)
end
```

With the ActsAsTaggableOn gem, every model class responds to acts_as_taggable and, when called for the second half of the boolean statement, actually makes the class taggable. I could be misunderstanding the scope here, but this is what seems to be happening.

**Comment by arnaudbesnier** [comment]
@romanos thanks for the feedback!
The fixed has been released in the latest gem version (v2.4.8).

🌲🌲🌲

**Comment by romanos** [comment]
Thanks!
